mule-apikit-tools/mule-apikit-scaffolder/src/test/java/org/mule/tools/apikit/ScaffolderWithExistingConfigMule4Test.java

```diff
@@ -6,38 +6,70 @@
  */
 package org.mule.tools.apikit;
 
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.apache.maven.plugin.logging.Log;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.mule.raml.implv2.ParserV2Utils;
+import org.mule.tools.apikit.misc.FileListUtils;
+
 import java.io.File;
 import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.List;
+import java.util.Map;
 import java.util.Set;
 import java.util.TreeSet;
-import org.apache.commons.io.FileUtils;
-import org.apache.commons.io.IOUtils;
-import org.junit.Test;
-import org.mule.raml.implv2.ParserV2Utils;
 
-import static java.util.Collections.emptyList;
 import static java.util.Collections.singletonList;
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
 import static org.mule.tools.apikit.Helper.countOccurences;
-
-public class ScaffolderWithExistingConfigMule4Test extends AbstractScaffolderTestCase {
+import static org.mule.tools.apikit.Scaffolder.DEFAULT_MULE_VERSION;
+import static org.mule.tools.apikit.Scaffolder.DEFAULT_RUNTIME_EDITION;
+
+public class ScaffolderWithExistingConfigMule4Test {
+
+  @Rule
+  public TemporaryFolder folder = new TemporaryFolder();
+  private FileListUtils fileListUtils = new FileListUtils();
+
+  @Before
+  public void setUp() throws IOException {
```

Comment:
All test in this class was improved and simplified before APISync. Please, try to merge only minimal changes

Reply:
Will do